src/ListTasks1/OneLinkedList.c

```diff
@@ -0,0 +1,117 @@
+#include "OneLinkedList.h"
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+typedef struct ListNode ListNode;
+
+struct ListNode {
+    int value;
+    struct ListNode* next;
+};
```

Comment:
это то же самое, что и
```suggestion
typedef struct ListNode {
    int value;
    struct ListNode* next;
} ListNode;
```
но так тоже можно